assets/style/app/components/_page-footer.scss

```diff
@@ -65,6 +65,9 @@
       margin-bottom: $space-base;
     }
 
+    .sub-menu {
+      font-size: 12px !important;
```

Comment:
is there a way you can nest this style so that `!important` isn't used? this is really a last resort